yt_dlp/options.py

```diff
@@ -1421,7 +1422,7 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help='Do not use .part files - write directly into output file')
     filesystem.add_option(
         '--mtime',
-        action='store_true', dest='updatetime', default=True,
+        action='store_true', dest='updatetime', default=None,
         help='Use the Last-modified header to set the file modification time (default)')
```

Comment:
need to move ` (default)` to `--no-mtime`
```suggestion
        help='Use the Last-modified header to set the file modification time')
```